integration/token/fungible/views/history.go

```diff
@@ -92,3 +90,51 @@ func (i *ListAuditedTransactionsViewFactory) NewView(in []byte) (view.View, erro
 	assert.NoError(err, "failed unmarshalling input")
 	return f, nil
 }
+
+// ListAcceptedTransactions contains the input to query the list of issued tokens
```

Comment:
ListAcceptedTransactions contains the input to query the list of accepted tokens?